### PR TITLE
Split Bio-Formats C++ from bioformats.git

### PR DESCRIPTION
**Pull request by ghost**
- Add biofomats-cpp package
- Rename messages to reflect old and new repository contents

---

See https://github.com/openmicroscopy/bioformats/pull/2258 for testing instructions

### CODE REVIEW

File: packages/bioformats/superbuild.cmake [hunk 1]
```diff
@@ -28,20 +28,20 @@ if(bf-dir)
     DOWNLOAD_COMMAND "")
   set(EP_SOURCE_DIR "${bf-dir}")
   set(BOOST_VERSION 1.60)
-  message(STATUS "Building Bio-Formats from local directory (${bf-dir})")
+  message(STATUS "Building OME XML C++ from local directory (${bf-dir})")
```
Comment (joshmoore) [inline packages/bioformats/superbuild.cmake]: Is this from https://github.com/ome/ome-common-cpp? It might just be slipping my memory what this is.

Comment (ghost) [inline packages/bioformats/superbuild.cmake]: It's updating the message to reflect that ome-xml is the only component remaining here now the rest is split out.

Comment (joshmoore) [inline packages/bioformats/superbuild.cmake]: Aha. So "OME-XML" as the only C++ product from openmicroscopy/bioformats, right?

Comment (ghost) [inline packages/bioformats/superbuild.cmake]: Yes.

Comment (sbesson) [inline packages/bioformats/superbuild.cmake]: This naming choice is certainly in line with the Java `ome-xml.jar` package name.

**Comment by ghost** [comment]
See also https://github.com/ome/ome-files-cpp/pull/1

Green in https://ci.openmicroscopy.org/view/DEV/job/BIOFORMATS-CPP-DEV-merge/90/ and https://ci.openmicroscopy.org/view/DEV/job/BIOFORMATS-CPP-DEV-merge-superbuild/116/ and https://ci.openmicroscopy.org/view/DEV/job/BIOFORMATS-CPP-DEV-merge-win-superbuild/133/

**Comment by sbesson** [comment]
Thanks, this indeed takes care of most of the renaming and gets all jobs green. Merging.

**Pull request merged**